scapy/contrib/http2.py

```diff
@@ -638,14 +636,13 @@ def _compute_value(self, pkt):
 # Welcome the magic of Python inconsistencies !
 # https://stackoverflow.com/a/41622155
 
-
 if sys.version_info >= (3, 4):
     ABC = abc.ABC
 else:
     ABC = abc.ABCMeta('ABC', (), {})
 
 
-class HPackStringsInterface(ABC, Sized):
+class HPackStringsInterface(ABC, Sized):  # type: ignore
```

Comment:
I've used `# type: ignore` only in here, but that's very messy and hard to fix.
`abc.ABC` is already super tricky to get working (see the tweaks above)